MWContentDisplayPlugin/MWContentDisplayPlugin/ContentStackStep/SwiftUIViews/MWStackView.swift

```diff
@@ -109,6 +109,7 @@ fileprivate struct MWListItemView: View {
     }
 }
 
+#warning("Button height is currently set to 44pt to match RK Button. Once we get rid of Research Kit, we'll provide the same height (as per design guidelines 50pt) in both Core and Plugins.")
```

Comment:
@FWJonathan Added it as a warning, so we don't miss it out once buttons are redesigned (after RK is gone)